plugin/kakadujp2/src/test/java/it/geosolutions/imageio/plugins/jp2k/JP2KKakaduWriteTest.java

```diff
@@ -39,6 +39,7 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.util.LinkedList;
+import java.util.Locale;
```

Comment:
Forgotten import not used anywhere (I think when testing the floating point parsing)

Reply:
Removing